core/src/main/scala/cats/std/anyval.scala

```diff
@@ -53,90 +52,80 @@ trait CharInstances /* missing algebra type classes */ {
+    new Order[Short] {
       def compare(x: Short, y: Short): Int =
         if (x < y) -1 else if (y < x) 1 else 0
     }
 
 }
 
-trait LongInstances /* missing algebra type classes */ {
+trait LongInstances {
 
   implicit val longShow: Show[Long] =
     Show.fromToString[Long]
 
-  // TODO: replace this minimal algebra with one from the algebra project
-  implicit val longAlgebra: CommutativeGroup[Long] with Order[Long] =
-    new CommutativeGroup[Long] with Order[Long] {
-      def combine(x: Long, y: Long): Long = x + y
+  implicit val longAlgebra: Group[Long] with Order[Long] =
+    new Group[Long] with Order[Long] {
       def empty: Long = 0L
-      def inverse(x: Long): Long = -x
+      def inverse(a: Long): Long = -a
+      def combine(x: Long, y: Long): Long = x+y
       def compare(x: Long, y: Long): Int =
         if (x < y) -1 else if (y < x) 1 else 0
     }
 }
 
-trait FloatInstances /* missing algebra type classes */ {
+trait FloatInstances {
 
   implicit val floatShow: Show[Float] =
     Show.fromToString[Float]
 
-  // TODO: replace this minimal algebra with one from the algebra project
-  implicit val floatAlgebra: CommutativeGroup[Float] with Order[Float] =
-    new CommutativeGroup[Float] with Order[Float] {
-      def combine(x: Float, y: Float): Float = x + y
-      def empty: Float = 0F
-      def inverse(x: Float): Float = -x
+  implicit val floatOrder: Order[Float] =
+    new Order[Float] {
       def compare(x: Float, y: Float): Int =
         java.lang.Float.compare(x, y)
     }
 
 }
 
-trait DoubleInstances /* missing algebra type classes */ {
+trait DoubleInstances {
 
   implicit val doubleShow: Show[Double] =
     Show.fromToString[Double]
 
-  // TODO: replace this minimal algebra with one from the algebra project
-  implicit val doubleAlgebra: CommutativeGroup[Double] with Order[Double] =
-    new CommutativeGroup[Double] with Order[Double] {
-      def combine(x: Double, y: Double): Double = x + y
-      def empty: Double = 0D
-      def inverse(x: Double): Double = -x
+  implicit val doubleOrder: Order[Double] =
+    new Order[Double] {
       def compare(x: Double, y: Double): Int =
         java.lang.Double.compare(x, y)
     }
 
 }
 
-trait BooleanInstances extends algebra.std.BooleanInstances {
+trait BooleanInstances {
 
   implicit val booleanShow: Show[Boolean] =
     Show.fromToString[Boolean]
 
+  implicit val booleanOrder: Order[Boolean] =
+    new Order[Boolean] {
+      def compare(x: Boolean, y: Boolean): Int =
+        if (x == y) 0 else if (x) 1 else -1
+    }
+
 }
 
-trait UnitInstances /* missing algebra type classes */ {
+trait UnitInstances {
 
   implicit val unitShow: Show[Unit] =
     Show.fromToString[Unit]
 
-  implicit val unitAlgebra: CommutativeGroup[Unit] with Order[Unit] =
-    new CommutativeGroup[Unit] with Order[Unit] {
-      def combine(x: Unit, y: Unit): Unit = ()
-      def empty: Unit = ()
-      def inverse(x: Unit): Unit = ()
+  implicit val unitOrder: Order[Unit] =
```

Comment:
Yes, sorry. I overlooked that you just removed the CommutativeGroup, but that's another issue to keep in mind with standard instances: many standard ones are commutative, but kernel won't have it, so something for algebra to keep in mind.